data/json/recipes/weapon/piercing.json

```diff
@@ -1,4 +1,101 @@
 [
+  {
+    "result": "fencing_epee",
```

Comment:
The fencing weapons need electrical components as well since they're electronic scoring types.

Reply:
Pretty much only cables and a tiny wee spring for the epee and foil.

The sabers have a electrical socket, basically no other components.

Reply:
Changed fencing weapons recipes to create their sharpened "combat" versions instead (as they are explicitly stated to come with these components removed in description), so that player wouldn't have to bother with electronic components (and they're more useful than unsharpened versions anyway)